### PR TITLE
Add a web UI for monitoring Apache Kafka clusters

### PR DESCRIPTION
**Pull request by PauloPortugal**
Kafdrop is a web UI for monitoring Apache Kafka clusters. The tool
displays information such as brokers, topics, partitions, consumers
(including lag) and lets you view messages on our docker network.
Screenshot from 2022-05-04 17-52-47

### CODE REVIEW

**Review by j-s-rawat — COMMENTED** [review]
LGTM

**Review by j-s-rawat — CHANGES_REQUESTED** [review]
I tried but the it failed, so we might have to fix or look at the issue before merge

File: cantabular-import/deps.yml [hunk 1]
```diff
@@ -88,3 +88,14 @@ services:
     environment:
       - VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200
       - VAULT_DEV_ROOT_TOKEN_ID=0000-0000-0000-0000
+  kafdrop:
+    image: 'obsidiandynamics/kafdrop'
+    restart: "no"
+    ports:
+      - "9090:9000"
+    environment:
+      KAFKA_BROKERCONNECT: "kafka-1:19092"
```
Comment (j-s-rawat) [inline cantabular-import/deps.yml:97]: Adding the below line in environment after KAFKA_BROJERCONNECT fixed the issue for kafdrop to work
`JVM_OPTS: "-Xms16M -Xmx48M -Xss180K -XX:-TieredCompilation -XX:+UseStringDeduplication -noverify"`

Comment (PauloPortugal) [reply cantabular-import/deps.yml:97]: This has been addressed.

**Review by j-s-rawat — APPROVED** [review]
LGTM

**Review by j-s-rawat — APPROVED** [review]

**Pull request merged**